### ISSUE TITLE
Error when running npx init

### ISSUE THREAD

**Issue by scottgallant** [tinacms/tinacms#1938]
## Description

I'm getting the following error when running `npx @tinacms/cli init`

```
npx @tinacms/cli init
npm ERR! cb.apply is not a function

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/scott/.npm/_logs/2021-07-29T01_48_20_318Z-debug.log
Install for [ '@tinacms/cli@latest' ] failed with code 1

```

### Environment

I'm running Node v14.17.0.

When I switch to Node v 16.x there is no issue. Does this require Node 16.x? 

#### Tinacms Package Versions

```
 npm ls | grep tinacms | grep -v deduped
npm ERR! code ELSPROBLEMS
npm ERR! invalid: next@11.0.1 /Users/scott/Documents/projects/tina/starter/tina-cloud-starters/tina-quickstart/node_modules/next

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/scott/.npm/_logs/2021-07-29T02_06_12_007Z-debug.log
├── @tinacms/cli@0.50.1
└── tinacms@0.50.1
```

**Comment by DirtyF** [comment]
Just ran init command on my computer with the same Node version, and it works as expected: 

```
blog-starter-tinacms on main via ⬢ v14.17.0
npx @tinacms/cli init
Need to install the following packages:
  @tinacms/cli
Ok to proceed? (y) y
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated axios@0.19.0: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410

Setting up Tina...
Installing Tina packages. This might take a moment... ✅
✔ do you want us to override your _app.js? … yes
Adding _app.js ... ✅
Adding a content folder... ✅
Tina setup done  ✅
	 Start your dev server with yarn tina-dev and go to http://localhost:3000/demo/blog/HelloWorld to check it out the page that was created for you
Enjoy Tina 🦙 !
```

**Comment by logan-anderson** [comment]
@scottgallant this might be an error with Node.js itself.

[here](https://github.com/nodejs/help/issues/2874) is the issue and [here](https://github.com/nodejs/help/issues/2874#issuecomment-663661148) is a workaround on how to fix it.
